### PR TITLE
Improve Chinese translation

### PR DESCRIPTION
**Pull request by nijinekoyo**
Improved Chinese translation, new language files translated

### CODE REVIEW

File: iTorrent/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -304,7 +304,7 @@
 "Peers" = "用户";
 "Seeds" = "做种";
 "Leechers" = "下载";
-"Are you shure to remove this trackers?" = "你确定要删除这个 Tracker 吗？";
```
Comment (XITRIX) [inline iTorrent/zh-Hans.lproj/Localizable.strings:307]: Do not fix localization keys without replacing them in all translation files and project itself

Comment (nijinekoyo) [reply iTorrent/zh-Hans.lproj/Localizable.strings:307]: I noticed that the English translation file modified this Key, and this string was indeed rendered untranslated in the program.

Comment (nijinekoyo) [reply iTorrent/zh-Hans.lproj/Localizable.strings:307]: IMG_2506

Comment (nijinekoyo) [reply iTorrent/zh-Hans.lproj/Localizable.strings:307]: The modifications to the English translation are located in this commit
https://github.com/XITRIX/iTorrent/commit/e9a4d3ecd9be2f93d690f581d56fba7cc0c80d5f

Comment (XITRIX) [reply iTorrent/zh-Hans.lproj/Localizable.strings:307]: Ah, sorry for that, you are right

**Comment by XITRIX** [comment]
Thanks for your contribution!

**Comment by rmtkcjs** [comment]
gg

**Review by rmtkcjs — COMMENTED** [review]
gg

**Comment by nijinekoyo** [comment]
?

**Pull request merged**